doc/src/manual/noteworthy-differences.md

```diff
@@ -167,13 +169,14 @@ For users coming to Julia from R, these are some noteworthy differences:
   * Julia's [`sum`](@ref), [`prod`](@ref), [`maximum`](@ref), and [`minimum`](@ref) are different
     from their counterparts in R. They all accept one or two arguments. The first argument is an iterable
     collection such as an array.  If there is a second argument, then this argument indicates the
-    dimensions, over which the operation is carried out.  For instance, let `A=[[1 2],[3 4]]` in Julia
-    and `B=rbind(c(1,2),c(3,4))` be the same matrix in R.  Then `sum(A)` gives the same result as
-    `sum(B)`, but `sum(A, 1)` is a row vector containing the sum over each column and `sum(A, 2)`
+    dimensions, over which the operation is carried out.  For instance, let `A=[1 2; 3 4]` in Julia
+    and `B <- rbind(c(1,2),c(3,4))` be the same matrix in R.  Then `sum(A)` gives the same result as
+    `sum(B)`, but `sum(A, dims=1)` is a row vector containing the sum over each column and `sum(A, dims=2)`
     is a column vector containing the sum over each row.  This contrasts to the behavior of R, where
-    `sum(B,1)=11` and `sum(B,2)=12`.  If the second argument is a vector, then it specifies all the
-    dimensions over which the sum is performed, e.g., `sum(A,[1,2])=10`.  It should be noted that
-    there is no error checking regarding the second argument.
+    `sum(B,1) == 11` and `sum(B,2) == 12`.  If `dims` keyword argument is a vector, then it specifies all
```

Comment:
Missing `dims=`?

Reply:
Also missing a "the": "If *the* \`dims\` keyword argument..."

Reply:
Both fixed. In R I decided to replace the examples `colSums` and `rowSums`  functions as they are designed to make such sums so it is a better reference.